webpay/api/tests/test_api.py

```diff
@@ -132,6 +133,12 @@ def test_configures_transaction_fail(self):
         res = self.post(req='')  # cause a form error.
         eq_(res.status_code, 400)
 
+    def test_unsupported_jwt_algorithm(self):
+        with self.settings(SUPPORTED_JWT_ALGORITHMS=['HS256']):
+            res = self.post(
+                request_kwargs={'jwt_kwargs': {'algorithm': 'none'}})
```

Comment:
``{algorithm: none}`` is just a keyword sent to the JWT library which produces a JWT with the standard ``alg`` field. I did see the test fail before the patch so it should be working.